Common/ConfigParser.cs

```diff
@@ -21,9 +21,9 @@ public class YamlConfig
             public List<ProcessSection> Match { get; set; }
 
             /// <summary>
-            /// Whether SSL validation should be globally disabled. Null if the user did not set this property.
+            /// Whether SSL validation should be globally enabled. Null if the user did not set this property.
             /// </summary>
-            public bool? DisableSslValidation { get; set; }
+            public bool? EnableSslValidation { get; set; }
```

Comment:
This is a breaking change of the config format. We'll have to change all customer configs that are used right now. I'd like to avoid this and keep the name of the option as is if possible

Reply:
Good point, I didn't think of this. Then I'll change back all variable names in the code, so that we don't have both variants which actually adds confusion.